pep-0007.txt

```diff
@@ -35,6 +35,9 @@ C dialect
   <https://en.wikipedia.org/wiki/C11_%28C_standard_revision%29#Optional_features>`_.
   The public C API should be compatible with C++.
 
+* Python versions up to and including 3.11 tacitly allowed signed arithmetic
+  overflow (wrapping around using twos-complement representation) via the `-fwrapv` compiler option.
```

Comment:
Double backticks for RST (this is RST even though it has a `.txt` extension):
```suggestion
  overflow (wrapping around using twos-complement representation) via the ``-fwrapv`` compiler option.
```

Reply:
Thanks!